Add conda-forge and pypi badges + update installation instructions (#149)
COSIMA/mom6-regional-scripts: README.md

## Installation

> #### Easy, clean, one liner

Reviewer: This and a couple below, maybe don't skip a header level?

```suggestion
### Easy, clean, one liner
```

Author: it was just a bit too big with `###` that's why I put 4 `####`

Reviewer: Fair enough!

The easiest way is to install `regional-mom6` via [`conda`](https://anaconda.org/conda-forge/regional-mom6).
We encourage creating a new or using an existing conda environment and then simply

```bash
conda install conda-forge::regional-mom6
```

That's it -- now enjoy!

> #### "*But I want `pip`, can't I install with `pip`*?"

Reviewer: ```suggestion
### "*But I want `pip`, can't I install with `pip`*?"
```

With `esmpy` available, we can then install `regional_mom6` via pip. (If we don't have have pip, then
`conda install pip` should do the job.)

> With `esmpy` install we can now install `regional-mom6` via [`pip`](https://badge.fury.io/py/regional-mom6):

Reviewer: ```suggestion
With `esmpy` installed we can now install `regional-mom6` via [`pip`](https://pypi.org/project/regional-mom6/):
```

```bash
pip install regional-mom6
```

The above installs the version of `regional-mom6` (plus any required dependencies) that corresponds to the latest tagged release of the package.

> #### "*I'd like to be on the cutting edge of the development*?"

Reviewer: ```suggestion
### "*I'd like to be on the cutting edge of the development*?"
```

> Alternatively, we can install directly `regional-mom6` directly via Github using `pip`.

Reviewer: ```suggestion
Alternatively, we can install directly `regional-mom6` directly via GitHub using `pip`.
```

First install `esmpy` as described above and then:

```bash
pip install git+https://github.com/COSIMA/regional-mom6.git
```

to get the version that corresponds to the latest commit in GitHub.

> Or, install the version that corresponds to a particular git commit via

Reviewer: Maybe make it clearer that this is an example commit?

Also **super** minor nit: you say "via GitHub using" above, but here you say "via", maybe should be "using" here instead?

Author: done
